inputs/complex-list-item.vue

```diff
@@ -109,14 +7,14 @@
           <div class="complex-list-item-actions">
             <div class="complex-list-item-actions-inner ui-button-group">
               <div class="complex-list-item-actions-left ui-button-group">
-                <ui-icon-button v-if="!isFirstItem && !isFiltered" buttonType="button" type="secondary" color="black" icon="keyboard_arrow_up" @click.stop.prevent="moveItem(originalIndex, 'up')"></ui-icon-button>
+                <ui-icon-button v-if="!isFirstItem && !isFiltered" buttonType="button" type="secondary" color="black" icon="keyboard_arrow_up" @click.stop.prevent="$emit('moveItem', {index: originalIndex, direction: 'up'})"></ui-icon-button>
                 <span class="complex-list-item-position">{{ originalIndex + 1 }}/{{ total }}</span>
-                <ui-icon-button v-if="!isLastItem && !isFiltered" buttonType="button" type="secondary" color="black" icon="keyboard_arrow_down" @click.stop.prevent="moveItem(originalIndex, 'down')"></ui-icon-button>
+                <ui-icon-button v-if="!isLastItem && !isFiltered" buttonType="button" type="secondary" color="black" icon="keyboard_arrow_down" @click.stop.prevent="$emit('moveItem', {index: originalIndex, direction: 'down'})"></ui-icon-button>
```

Comment:
You could have a function to reuse some code

```suggestion
                <ui-icon-button v-if="!isLastItem && !isFiltered" buttonType="button" type="secondary" color="black" icon="keyboard_arrow_down" @click.stop.prevent="moveItem('down')"></ui-icon-button>
```
Also this function will be used [here](https://github.com/clay/clay-kiln/pull/1405/files#diff-292123298ccaf38109e32dddbadc0d01R10)


```js
methods: {
  moveItem(direction) {
    this.$emit('moveItem', { index: this.originalIndex, direction })
  }
}
```

Reply:
👍